modules/get-started/pages/whats-new-241.adoc

```diff
@@ -0,0 +1,28 @@
+= What's New in 24.1 Beta
+:description: Summary of new features and updates in the release.
+
+This topic includes new content added in version 24.1 beta. 
+
+== Write caching
+
+xref:develop:config-topics/#configure-write-caching[Write caching] is a relaxed mode of `acks=all` that provides better performance at the expense of durability. It acknowledges a message as soon as it is received and acknowledged on a majority of brokers, without waiting for it to fsync to disk. This provides lower latency while still ensuring that a majority of brokers acknowledge the write. For clusters in development mode, write caching is enabled by default. For clusters in production mode, it is disabled by default.
+
+== RBAC
+xref:security:authorization/rbac.adoc[Role-based access control] (RBAC) provides an extension to Redpanda's existing access control lists (ACLs). It allows you to create roles and assign those roles as the authorized or denied party for an ACL. Every user you assign to that role inherits those authorizations. Using RBAC gives you a more flexible tool to manage authorization for your users, especially when you have complex organizational hierarchies, large numbers of users, or both.
+
+== Data transforms
+
+* Output to multiple topics: Data transforms can now output to multiple topics, for use cases that might require distributing different message types to multiple event streams, or for writing valid records to an output topic and invalid records to a dead-letter queue. See the xref:https://docs.redpanda.com/current/reference/data-transform-golang-sdk/[Go] or xref:https://docs.redpanda.com/current/reference/data-transform-rust-sdk/[Rust] SDK reference for data transforms for implementation details.
+
+: Transform logs stored in topic xref:develop/data-transforms/run-transforms.adoc#view-data-transform-logs[Data transform logs] are now stored in an internally-managed topic.
+
+== New commands
+
+New commands and properties in this release include the following:
+
+- xref:reference:rpk/rpk-registry/rpk-registry.adoc[`rpk registry`]: This includes commands to interact with the Schema Registry.
+- xref:reference:rpk/rpk-transform/rpk-transform.adoc[`rpk transform`]: This includes commands to develop, deploy, and manage data transforms.
+
+== Next steps
+
+* xref:get-started:install-beta.adoc[]
```

Comment:
Title will change to `Install 24.1 beta` once we merge https://github.com/redpanda-data/docs/pull/402